CHANGELOG.md

```diff
@@ -2,6 +2,99 @@
 
 ## 5.14.8
 
+<!-- generated comparing v5.14.8..master -->
+
+_Sep 11, 2023_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎉 Added `Drawer` component to Joy UI (#38169) @mnajdova
+- ✨ Added `firstButton`, `lastButton`, and `middleButton` classes to `ButtonGroup`'s button descendants in Material UI (#38520) @ZeeshanTamboli
```

Comment:
This change is unclear to me, what's the problem solved? Or maybe it's not worth a highlight.

Reply:
This is a quality-of-life improvement for users of the `ButtonGroup` component, as they can now target the buttons using classes.

What do you think about

```suggestion
- ✨ The Material UI `ButtonGroup` now provides utility classes for its buttons' position (#38520) @ZeeshanTamboli
```

@ZeeshanTamboli I took inspiration from your suggestion above, but I would like to highlight it positively ("We're adding this quality-of-life enhancement") instead of negatively ("This is now working correctly")

Reply:
The main aim for the PR was not provide these classes for buttons, but to fix the styling for:
- buttons with hrefs - #31210
- disabled buttons displaying tooltips - #29514

Looking from a user's perspective, I wouldn't be particularly enthusiastic about classes available (especially in a highlight), but on understanding what is the problem solved using those classes. Yes, you can use the classes for customization but that was not the goal of the PR.

Reply:
@ZeeshanTamboli ok, that makes sense, I updated it 🙌🏼

Reply:
I think we could make the case that #38520 is a breaking change.